config/application.rb

```diff
@@ -14,6 +14,9 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.0
```

Comment:
Do we need 5.2 here?
```suggestion
    config.load_defaults 5.2
```

Reply:
yes totally!